### PR TITLE
Update metacache to 2.0.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`metacache`](https://bioconda.github.io/recipes/metacache/README.html): **1.1.1** &rarr; **2.0.0**

[install with bioconda](http://bioconda.github.io/recipes/metacache/README.html) [Conda](https://anaconda.org/bioconda/metacache/files)

Info | Link
-----|-----
Recipe | [`recipes/metacache`](https://github.com//bioconda/bioconda-recipes/tree/bump/metacache/recipes/metacache) (click to view/edit other files)
Releases |[https://github.com/muellan/metacache/tags](https://github.com/muellan/metacache/tags)
Author | `@muellan`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/metacache/meta.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ requirements:
   host:
     - gawk
     - wget
+    - zlib
```
Comment (fanninpm) [inline recipes/metacache/meta.yaml:22]: `zlib` needs to be a build dependency. Remind me later to do that.

Comment (bgruening) [reply recipes/metacache/meta.yaml:22]: no, host is ok. It should not be in build and can be skipped in run.

Comment (fanninpm) [reply recipes/metacache/meta.yaml:22]: What explains the CI failure?

Comment (druvus) [reply recipes/metacache/meta.yaml:22]: You could try this: https://bioconda.github.io/contributor/troubleshooting.html#zlib-errors

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Pull request closed without merging**